Cargo.toml

```diff
@@ -9,15 +9,15 @@ keywords      = ["rpm", "linux", "redhat", "fedora", "centos"]
 readme        = "README.md"
 homepage      = "https://github.com/rpm-software-management/librpm.rs/"
 documentation = "https://rustrpm.org/librpm/"
-edition       = "2018"
+edition       = "2021"
```

Comment:
I'm okay with an edition bump, but IMO it should be done in a separate PR that notes the corresponding MSRV change